amazon-kinesis-client/src/test/java/software/amazon/kinesis/common/StreamIdentifierTest.java

```diff
@@ -44,7 +44,8 @@ public void testMultiStreamDeserializationFail() {
                 "123456789012:stream-name:", // missing creation epoch
                 "123456789012:stream-name:-123", // negative creation epoch
                 "123456789012:stream-name:abc", // non-numeric creation epoch
-                ""
+                "",
+                "::" // missing account id, stream name, and epoch
```

Comment:
FYI: you could have inserted this after line ~38 to avoid the unnecessary `s/""/"",/` edit. Always appending to the tail:
1. may unnecessarily increase the blast radius of the edit (unless there's a pragmatic reason like ordering)
1. increases the chance of merge conflicts (e.g., two+ contributors append to tail; likely everyone but the first-to-merge suffers a conflict)